cmd/inspect.go

```diff
@@ -112,37 +110,24 @@ func getInspectCmd(logger *logrus.Logger, globalFlags *commandFlags) *cobra.Comm
 	return inspectCmd
 }
 
-func addExecRequirements(b *js.Bundle,
-	builtinMetrics *metrics.BuiltinMetrics, registry *metrics.Registry,
-	logger *logrus.Logger, globalFlags *commandFlags) (interface{}, error) {
-	// TODO: after #1048 issue, consider rewriting this without a Runner:
-	// just creating ExecutionPlan directly from validated options
-
-	runner, err := js.NewFromBundle(logger, b, builtinMetrics, registry)
-	if err != nil {
-		return nil, err
-	}
-
+func addExecRequirements(b *js.Bundle, logger *logrus.Logger, globalFlags *commandFlags) (interface{}, error) {
```

Comment:
I would suggest taking the opportunity to document the function. Moreover, maybe we could find a more suitable name? In general, something is **add**ed **to** something, but in this case, the function _produces_ something instead?

Reply:
Also I've spotted an inconsistency in the naming of arguments: `b` is shorthand for its type:`Bundle`, but `logger` is the full name of its type: `Logger`. In this case, I would plead for renaming `b` to `bundle` for clarity, but I'm not hung up on either one or the other solution.

Reply:
Good points :+1: The one about `bundle` was already addressed in latter PR by just not needing it anymore :sweat_smile: https://github.com/grafana/k6/blob/691be6b68ef829a0ccf514220d91d06befb2bc84/cmd/inspect.go#L79

About the rest, for purely practical purposes, I'd prefer to also fix them in a separate commit/PR and leave this PR for only the actual code simplification.